internal/test_programs/s2/2.lox

```diff
@@ -0,0 +1,2 @@
+print <<RANDOM_QUOTEDSTRING>>;
+print <<RANDOM_BOOLEAN>>;
```

Comment:
Merge this into previous stage - we need randomness in every stage, not a separate test for randomness